cfy_cluster_manager/main.py

```diff
@@ -1061,6 +1126,22 @@ def main():
     add_config_arg(remove_args)
     add_verbose_arg(remove_args)
 
+    upgrade_args = subparsers.add_parser(
+        'upgrade',
+        help='Upgrade a v5.1.0 Cloudify cluster to v5.1.1')
+
+    add_config_arg(upgrade_args)
+    upgrade_args.add_argument(
+        '--upgrade-rpm',
+        action='store',
+        default=UPGRADE_RPM_PATH,
+        help='Path to a v5.1.1 cloudify-manager-install RPM. '
```

Comment:
That's a great comment. I also thought we should make this more generic, but then I came to the conclusion that we don't know yet how the upgrade process from 5.1.1 to 5.1.2 would look like (maybe we would even enable users to upgrade from 5.1.0 to 5.1.2). So I focused on the upgrade from v5.1.0 to 5.1.1 and will update to code once we release v5.1.2. Let me know if it makes sense.

Regarding the default 5.1.1 URL, if prefer to keep it and update it once we release v5.1.1 ga. This way we prevent user errors.

Reply:
I agree but in case of clients working in offline environment that would be an issue somehow if they use the default url. anyway we can for now have this on master and once we release the GA you are going to update it again to GA and also bump the version